hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -1971,6 +1971,52 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
+          "hederaFunctionality": "TokenClaimAirdrop",
```

Comment:
Probably same comment this should have fees same as CryptoTransfer

Reply:
Fixed 👍